src/oci/conmon.rs

```diff
@@ -0,0 +1,292 @@
+//! Interface to [conmon][0], the OCI container runtime monitor.
+//!
+//! [0]: https://github.com/containers/conmon
+
+#![allow(dead_code)]
```

Comment:
Is this required for public interfaces? seems like rust would ignore these by default.

Reply:
Right now not, but we can consider making it public later on. For now this dead code hints us that it's not used anywhere inside the library. We will later use it directly in the OCI container implementation:
https://github.com/cri-o/cri/blob/master/src/oci/container/local.rs